model/CHEM_DRV.F90

```diff
@@ -439,107 +439,142 @@ SUBROUTINE DO_CHEM
              if(hassouthpole(grid) .and. JJJ .eq. J_0 ) I = 1
              if(hasnorthpole(grid) .and. JJJ .eq. J_1 ) I = 1
 
+             ! Dry air mass [kg]
+             State_Met%AD          (II,JJ,K) = mma(i,j,k)
+
+             ! Dry air density [kg m-3]
+             ! NOTE: Formula from DIFFG function definition in model/DRYDEF.f with pressure PMID
+             State_Met%AIRDEN      (II,JJ,K) = PMID(II,JJ,K)*avog*bygasc/t(i,j,k)*pk(k,i,j)
+
+             ! Volume of grid box [m3]
+             ! TODO: State_Met%AIRVOL      (II,JJ,K) = ???
+             ! NOTE: We could calculate this using State_Grid%XEdge and State_Grid%YEdge but what
+             !       about the edge cases?
+
+             ! Water vapor mixing ratio (w/r/t dry air)
+             ! NOTE: QSAT is a function defined in model/shared/Utilities.F90
+             State_Met%AVGW        (II,JJ,K) = QSAT(t(i,j,k)*pk(k,i,j),LHE,pmid(k,i,j))
+
+             ! Grid box height [m]
+             ! NOTE: This is the geopotential height from Model E. Do we need to deduce the heights
+             !       from the GEOS-Chem State_Grid derived type?
+             State_Met%BXHEIGHT    (II,JJ,K) = gz(i,j,k)
```

Comment:
Review task: check these are the right arrays to use and suggest what to do for the case of `AIRVOL`.